Exiled.Events/EventArgs/Server/EndingRoundEventArgs.cs

```diff
@@ -52,7 +48,11 @@ public EndingRoundEventArgs(LeadingTeam leadingTeam, RoundSummary.SumInfo_ClassL
         /// <summary>
         ///     Gets or sets a value indicating whether the round is going to finish or not.
         /// </summary>
-        public bool IsRoundEnded { get; set; }
+        public bool IsRoundEnded
+        {
+            get => IsAllowed;
+            set => IsAllowed = value;
+        }
 
```

Comment:
Why is IsRoundEnded == to IsAllowed?

Reply:
because they are the same

Reply:
I would rather have IsRoundEnded than IsAllowed in this case.

Reply:
Done